### ISSUE TITLE
shopify create always shows "Authenticated Successfully, this page will close shortly" then show account not found error in console.

### ISSUE THREAD

**Issue by hedgerh** [Shopify/shopify-app-cli#436]
# Issue summary

`shopify create project` always shows me "Authenticated Successfully, this page will close shortly", followed by getting `✗ error: Your account was not found. Please sign up at https://partners.shopify.com/signup` in the console.

Tried removing entire `.shopify-app-cli` folder and reinstalling the shell command.  Tried removing `~/.shopify-app-cli/.tmp/.db.pstore`.   Tried clearing site data for the url.  Nothing seems to stop it from immediately going to "Authenticated Successfully, this page will close shortly."

## Expected behavior

After clearing data, it should open the browser and allow me to attempt to re-authenticate.

## Actual behavior

I'm immediately taken to `"Authenticated Successfully, this page will close shortly"` followed by `✗ error: Your account was not found. Please sign up at https://partners.shopify.com/signup` in the console.

## Steps to reproduce the problem

1. Run `shopify create project ae`
1. Enter credentials, potentially bad ones?
1. Observe console error
1. Remove `~/.shopify-app-cli/.tmp/.db.pstore` and try to repeat these steps

## Reduced test case

The best way to get your bug fixed is to provide a [reduced test case](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Reducing_testcases).

## Specifications

- App type: 
- Operating System:  OSX Catalina 10.15.2
- Shell: zsh
- Ruby version (ruby -v): ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]

**Comment by tanema** [comment]
Sorry, this might sounds like a stupid question but have you logged into the partner's dashboard before, and/or are you using the correct login. The authentication process just identifies you with our Shopify identity provider and then we use that identity to try to resolve your account in the partners dashboard. There is a possibility that you have another shopify account like a merchant account but have never signed into partners.

**Comment by hedgerh** [comment]
@tanema something really strange happened with my partners account.  I created one with an email address, changed that email address in settings, when I logged out and tried to log back in, the new email address had no partners account.  I was able to log in with the original email address, _but_ it did not have a partners account associated with it, but still was associated to my test store.

I went ahead and remade my partners account with the original email address, and now the CLI is working.

It's weird bc I've been working in the Partners dashboard on my app and with a test store all week..
